bin/wpc_init

```diff
@@ -121,7 +134,22 @@ set -e
 ## This code will be run during setup, INSIDE the container.
 ##
 
-wp core install --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title=Site
-# wp theme activate ...
-# wp plugin activate ...
+##############
+# Config
+##############
+title=your_site_title
```

Comment:
Hmm, there's something weird going on here. I can't get `wp core install` to accept a title containing spaces no matter how I do it - I always get a 'Too many positional arguments' error. Even putting the param in directly `wp core install --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title="A title with spaces"` gives the same error.

I kind find a few reports of similar behaviour, but nothing that makes me certain whether this is a `wp cli` bug or a problem with our docker image.